src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -1,14 +1,25 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project ToolsVersion="15.0">
   <Import Project="..\..\PowerShell.Common.props" />
+
   <PropertyGroup>
     <Description>PowerShell on Windows top-level project</Description>
     <AssemblyName>pwsh</AssemblyName>
     <OutputType>Exe</OutputType>
     <TieredCompilation>true</TieredCompilation>
+    <TieredCompilationQuickJit>true</TieredCompilationQuickJit>
+    <UseWindowsForms>false</UseWindowsForms>
+    <UseWPF>false</UseWPF>
```

Comment:
Since the value of these two properties is false, maybe we should just remove them?

Reply:
I suppose it would be cleaner without them.  Will remove.